holoviews/ipython/__init__.py

```diff
@@ -140,8 +139,21 @@ class notebook_extension(param.ParameterizedFunction):
 
     _loaded = False
 
+    _backends = {'matplotlib': 'mpl',
```

Comment:
Maybe a comment here would be good saying this is a dictionary from resource name to plotting subpackage import name.